feature-libs/organization/administration/components/unit/links/users/list/unit-user-link-cell.component.ts

```diff
@@ -32,11 +33,12 @@ export class UnitUserRolesCellComponent extends CellComponent {
   unitKey$: Observable<string> = this.itemService.key$;
   constructor(
     protected outlet: OutletContextData<TableDataOutletContext>,
-    protected itemService: ItemService<B2BUnit>
+    protected itemService: ItemService<B2BUnit>,
+    protected b2bUserService: B2BUserService
```

Comment:
This `UnitUserRolesCellComponent` component is a part of a feature library (not integration-lib). We are adding additional parameter `b2bUserService` to the constructor, which is not an optional one. Shouldn't we consider this as a breaking change if we plan to merge it into the next minor?

For develop branch this change is ok, as next release with a branch created from develop will be 6.0 (major).

Reply:
done